src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Arrays;
+import java.util.List;
+import model.FruitTransaction;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import service.ParserService;
+import service.impl.ParserServiceImpl;
+
+public class ParserServiceImplTest {
+    private static List<String> data;
+    private ParserService parserService = new ParserServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        data = List.of("b,banana,20",
+                "s,grape,20",
+                "p,banana,10");
+    }
```

Comment:
instantiate test data inside the test case. 

ParserServiceImpl should be instantiated inside BeforeAll.
```suggestion
    private static List<String> data;
    private ParserService parserService = new ParserServiceImpl();

    @BeforeAll
    static void beforeAll() {
        data = List.of("b,banana,20",
                "s,grape,20",
                "p,banana,10");
    }
```